help/markdown/fake-dotnet-testing-specflow.md

```diff
@@ -50,4 +52,41 @@ Target.create "Default" Target.DoNothing
 Target.runOrDefault "Default"
 ```
 
+## Minimal working example (v2.4+)
```

Comment:
I'd be ok with putting this above the `pre v2.4` docs but we can leave it as-is well.

Reply:
Good idea